packages/client/src/components/character-form.tsx

```diff
@@ -473,7 +567,21 @@ export default function CharacterForm({
 
           <Card>
             <CardContent className="p-6">
-              {CHARACTER_FORM_SCHEMA.map((section) => (
+              <div className="flex items-center gap-6 mb-6 pb-3 border-b">
+                <div className="flex items-center gap-2">
+                  <span className="inline-block w-3 h-3 rounded-full bg-red-500"></span>
+                  <span className="text-sm font-medium">Required</span>
+                </div>
+                <div className="flex items-center gap-2">
+                  <span className="inline-block w-3 h-3 rounded-full bg-yellow-500"></span>
+                  <span className="text-sm font-medium">Suggested</span>
+                </div>
+                <div className="flex items-center gap-2">
+                  <span className="inline-block w-3 h-3 rounded-full bg-gray-400"></span>
+                  <span className="text-sm font-medium">Optional</span>
+                </div>
+              </div>
```

Comment:
Let's remove this from the header and add a on hover tooltip for each icon saying the text